lib/state/appActions.js

```diff
@@ -817,4 +825,24 @@ appActions.reset = function(uploadIndex) {
   });
 };
 
+appActions.acknowledge = function (key) {
+  var update = localStore.getItem('acknowledgments');
+
+  if (!update) {
+    update = {};
+  }
+  
+
+  if (!update[key]) {
+    update[key] = {};
+  }
+
+  update[key][this.app.state.targetId] = Date.now();
```

Comment:
The Date.now() method returns the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC.